tutorials/3d/using_transforms.rst

```diff
@@ -9,7 +9,7 @@ Introduction
 If you have never made 3D games before, working with rotations in three dimensions can be confusing at first.
 Coming from 2D, the natural way of thinking is along the lines of *"Oh, it's just like rotating in 2D, except now rotations happen in X, Y and Z"*.
 
-At first this seems easy, and for simple games, this way of thinking may even be enough. Unfortunately, it's often incorrect.
+At first this seems easy, and for simple games this way of thinking may even be enough. Unfortunately, it's often incorrect.
```

Comment:
I suggest using shorter sentences. Long sentences usually don't read well in English.

```suggestion
At first, this seems easy. For simple games, this way of thinking may even be enough. Unfortunately, it's often incorrect.
```